src/main/java/com/erigitic/jobs/jobsets/FarmerJobSet.java

```diff
@@ -31,34 +31,34 @@
 
     private final String SETNAME = "crops";
 
-    private final String[][] REWARDS = {
-            //{"<event>", "<target>", "<expReward>", "<moneyReward>", "<growTrait>"}
+    private final Object[][] REWARDS = {
+            //{"<event>", "<target>", <expReward>, <moneyReward>, "<growTrait>"}
             // Plants having an age
-            {"break", "minecraft:wheat", "25", "50.00", "age"},
-            {"break", "minecraft:cocoa", "30", "60.00", "age"},
+            {"break", "minecraft:wheat", 25, 50.00d, "age"},
```

Comment:
Are the `d`'s necessary? It should default to a double if no type letter is specified.

Reply:
Yes they aren't absolutely necessary but I would use this just as the curly braces for single-statement ``if``s. To the untrained eye it's more clear that they're doubles.

Reply:
Fair point.